src/api_server/Cargo.toml

```diff
@@ -11,6 +11,7 @@ serde = { version = ">=1.0.27", features = ["derive"] }
 serde_derive = ">=1.0.27"
 serde_json = ">=1.0.9"
 derive_more = { version = "0.99.17", default-features = false, features = ["from"] }
+thiserror = "1.0.32"
```

Comment:
With `>=` we cannot guarantee an author won't make a change that breaks out build. `=` is the only way to fully gurantee this, we could loosen it with `= 1` instead if we trust they stick to SemVer.

I'm not strongly opinionated here, I'm not sure if we have a policy on this (we should probably make one if not). What do you think is best?

Reply:
Yes, qutie often. Even if they intend to abide SemVer (which they often don't) since their is no common automated checking mechanism (rust-vmm broke it recently). I have no strong opinion here, do you think we should go with `thiserror="1"`?